cmd/notation/cert/generateTest.go

```diff
@@ -51,6 +51,14 @@ func certGenerateTestCommand(opts *certGenerateTestOpts) *cobra.Command {
 			opts.name = args[0]
 			return nil
 		},
+		Long: `Manage certificates in trust store
+
+Example - Generate a test RSA key and a corresponding self-signed certificate named "wabbit-networks.io":
+  notation certificate generate-test "wabbit-networks.io"
```

Comment:
just FYI, this command also adds the cert into the trust store x509/ca/wabbit-networks.io.

Reply:
Thanks. This information will be told to users in the website (documentation).